src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2122 @@
+                ErrorCode error = VcMgrUnsetDialogRequestCb();
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Remove DialogRequested Failed with error " + error);
+                }
+
+                _dialogRequested -= value;
+                _dialogRequestDelegate = null;
+            }
+        }
+
+        /// <summary>
+        /// Enable command type as candidate command.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="cmdType">Command Type</param>
+        /// <exception cref="InvalidOperationException">This exception can be due to invalid state.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static void EnableCommandType(CommandType cmdType)
+        {
+            ErrorCode error = VcMgrEnableCommandType(cmdType);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "EnableCommandType Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Disable command type as candidate command.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="cmdType">Command Type</param>
+        /// <exception cref="InvalidOperationException">This exception can be due to invalid state.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static void DisableCommandType(CommandType cmdType)
+        {
+            ErrorCode error = VcMgrDisableCommandType(cmdType);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "DisableCommandType Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Called when engine sets private data to manager client.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        ///
```

Comment:
_privateDataRequestedDelegate  is always overwritted

Reply:
Do you need understand of multi handler of event

Reply:
Fixed.